### PR TITLE
KEYCLOAK-8530 KEYCLOAK-8531 Fix MigrationTest and migration from 2.5.…

### PR DESCRIPTION
**Pull request by mposolda**
…5.Final and 3.4.3.Final

### CODE REVIEW

File: testsuite/integration-arquillian/pom.xml [hunk 1]
```diff
@@ -73,6 +73,8 @@
         <migration.70.version>1.9.8.Final</migration.70.version>
         <migration.70.authz.version>2.2.1.Final</migration.70.authz.version>
```
Comment (vramik) [inline testsuite/integration-arquillian/pom.xml]: I believe this property and the profile `<id>test-70-authz-migration</id>` is no longer needed.

**Review by vramik — DISMISSED** [review]
@mposolda thank you, I have one small comment, but it can go in as it is.

**Comment by mposolda** [comment]
@vramik Thanks! I was thinking about removing it, and it the end I didn't, but will re-visit now.

**Comment by mposolda** [comment]
@vramik Updated

**Review by vramik — APPROVED** [review]
@mposolda thanks

**Review by pedroigor — APPROVED** [review]

**Pull request merged**